src/index.tsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-
+// import './assets/styles/GeneralStyles.scss'
+import './index.css'
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
+
 root.render(
-  <React.StrictMode>
+  // <React.StrictMode>
```

Comment:
It's best to remove commented code unless it's a `//TODO: ` or important notes for future authors
```suggestion
```

Reply:
I was testing how much the <React.StrictMode> was influencing. But I'll correct it